src/app/login/login.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="flex center ">
+  <mat-card class="mat-elevation-z8"> 
+    <mat-card-header>Notes App Login</mat-card-header>
+    <mat-card-content>
+      <form [formGroup]="loginForm" (ngSubmit)="login(this.loginForm)">
+        <mat-form-field appearance="standard">
+          <mat-label>User Name</mat-label>
+          <input matInput id="userName" name="userName" type="text" required formControlName="userName">
```

Comment:
If something is required, we'll usually put a <mat-error> for that requirement, same with password